src/lazy/encoder/binary/v1_0/container_writers.rs

```diff
@@ -338,4 +338,8 @@ impl<'value, 'top> StructWriter for BinaryStructWriter_1_0<'value, 'top> {
     fn close(self) -> IonResult<()> {
         self.container_writer.end()
     }
+
+    fn config(&self) -> ValueWriterConfig {
+        v1_0::Binary::default_value_writer_config()
+    }
```

Comment:
🗺️ The `StructWriter` impls need to be able to configure a value writer for each field. Only the application-level value writer types (`ListWriter`, `StructWriter`, etc) actually hold a `ValueWriterConfig` (at least for the time being), so the raw writer types just return the default `ValueWriterConfig` for their encoding.